### PR TITLE
Notes from Stephane in the prematurely-closed PR #120

### PR DESCRIPTION
**Pull request by paulehoffman**
Fixes to PR #120

### CODE REVIEW

**Comment by vttale** [comment]
It doesn't feel to me like "beyond what is already in {{RFC1035}}" really clarifies anything, and might just cause additional confusion, as it makes me as a reader wonder, "oh well what's in 1035?  Guess I have to go searching..."

**Comment by paulehoffman** [comment]
If you want to put the specific limit from RFC 1035 in, that's fine.

**Comment by vttale** [comment]
Well no, because I don't think there should be a limit.   While DNS messages implicitly have a maximum size of 64k *in DNS*, there's no reason dns-udpwireformat in HTTPS can't surpass that.

File: draft-ietf-doh-dns-over-https-latest.mkd [hunk 1]
```diff
@@ -191,7 +191,7 @@ wire format used in {{RFC7858}}.
 Also note that while {{RFC1035}} says "Messages carried by UDP are
 restricted to 512 bytes", that was later updated by {{RFC6891}},
 and this protocol does not put any limit on the size of DNS
-on-the-wire format payloads.
+on-the-wire format payloads beyond what is already in {{RFC1035}}.
```
Comment (martinthomson) [inline draft-ietf-doh-dns-over-https-latest.mkd]: The obvious reading of that is to substitute "what is already in 1035" with "restricted to 512 bytes", which I'm sure is not what you intend here.

I don't think that you need to make this change.  The only change I might consider is "put any **additional** limit on the size"

**Comment by paulehoffman** [comment]
@vttale: It would be an interoperability problem if a DNS API server handed a DNS API client a message that it could not handle. It is reasonable to assume that DNS API clients expect to parse messages as if they were DNS messages.

**Comment by paulehoffman** [comment]
@martinthomson: This was sloppy on my part: I meant the addition to point to RFC 6891; now changed.

**Comment by vttale** [comment]
Right @paulehoffman but my point is that something like multiple AXFR messages re-assembled into one unit is *still a DNS message* even if it exceeds 64k.   From a wire-format point of view, it parses exactly the same way.

**Comment by paulehoffman** [comment]
@vttale: Really? Ugh. Gotta love this protocol. 

So, you would prefer that the text says "...and this protocol allows DNS-on-the-wire format payloads of any size"?

**Comment by vttale** [comment]
Yes, I would.   Realistically almost everything is still going to be sub-10k, but I don't see an intrinsic need to limit it, especially with https channels routinely handling far larger responses.

**Comment by paulehoffman** [comment]
I think David and I are now in agreement; @mcmanus should consider accepting.

**Pull request merged**